packages/nextjs/app/builders/0x4a9A95B6fe3b9416f0c78A8735Aa075c75AF46a4/page.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
```

Comment:
I think we can remove "use client" since this is very much static page and removing "use client" gives use some nice optmization server components like SSR checkout [next docs](https://nextjs.org/docs/app/building-your-application/rendering/server-components#benefits-of-server-rendering)

Reply:
Removed 'useClient'